src/Repository/ConferenceRepository.php

```diff
@@ -33,6 +33,7 @@ public function findEndingCfps(): array
 
         return $this->createQueryBuilder('c')
             ->andWhere('c.cfpEndAt IS NOT NULL AND c.cfpEndAt >= :today AND c.cfpEndAt < :threshold')
+            ->andWhere('c.excluded = FALSE')
```

Comment:
```suggestion
            ->andWhere('c.excluded = :excluded')
            ->setParameter('excluded', false)
```

We let Doctrine casts the value itself